pkg/template/template.go

```diff
@@ -65,15 +65,16 @@ func (p *Processor) Process(template *api.Template) field.ErrorList {
 			item = decodedObj
 		}
 
-		newItem, err := p.SubstituteParameters(paramMap, item)
-		if err != nil {
-			templateErrors = append(templateErrors, field.Invalid(idxPath.Child("parameters"), template.Parameters, err.Error()))
-		}
 		// If an object definition's metadata includes a namespace field, the field will be stripped out of
 		// the definition during template instantiation.  This is necessary because all objects created during
 		// instantiation are placed into the target namespace, so it would be invalid for the object to declare
 		//a different namespace.
-		stripNamespace(newItem)
+		stripNamespace(item)
```

Comment:
Shouldn't either the comment above be tweaked, or a new comment added after this line, to equally reflect the new caveat of a parameter being able to override, even though the user still cannot explicitly set the namespace.